### PR TITLE
Hide previous parliament attendence review, prepare for new parliament

### PR DESCRIPTION
**Pull request by ebsuku**

### CODE REVIEW

File: pmg/models/resources.py [hunk 1]
```diff
@@ -1333,16 +1333,28 @@ def annual_attendance_trends(cls, from_year, to_year):
...
-        )\
-            .group_by('year', CommitteeMeeting.id)\
-            .filter(CommitteeMeeting.committee_id == committee_id)\
-            .filter(CommitteeMeetingAttendance.meeting_id == CommitteeMeeting.id)\
-            .subquery('attendance')
+    def committee_attendence_trends(cls, committee_id, period):
+        if period == 'current':
+            subquery = db.session.query(
+                func.date_part('year', CommitteeMeeting.date).label('year'),
+                func.count(case([(CommitteeMeetingAttendance.attendance.in_(CommitteeMeetingAttendance.ATTENDANCE_CODES_PRESENT), 1)])).label('n_present'),
```
Comment (jbothma) [inline pmg/models/resources.py]: thsi is all the same except for the date, right? Can't we assign the date to a variable and then have this code once?

Comment (ebsuku) [reply pmg/models/resources.py]: Not really, query is filtered differently based on whether is historical or not.

File: pmg/templates/committee_detail.html [hunk 2]
```diff
@@ -229,32 +229,55 @@ <h3 class="no-results">No results.</h3>
...
+              </div>
+		  </div>
+	  {% endfor %}
+	  Detailed attendance data by <a href="/attendance-overview">committee</a> and <a href="http://www.pa.org.za/mp-attendance/">MP</a>
+	  {% else %}
+		  <p>No current attendance data</p>
+	  {% endif %}
+            </div>
+	    <div class="panel-footer">
+		<a href="#" data-toggle="collapse" data-target="#historical">Click Here</a> to view the previous parliaments attendance data
```
Comment (jbothma) [inline pmg/templates/committee_detail.html]:
```suggestion
		<a href="#" data-toggle="collapse" data-target="#historical">Click Here</a> to view the previous parliament's attendance data
```

File: tests/unit/test_attendance.py [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+        attendance_two_mike = CommitteeMeetingAttendance(
+            attendance='A',
+            member=mike,
+            meeting=new_parliament_meeting,
+            created_at='2019-08-01')
+        db.session.add(attendance_two_mike)
+        db.session.commit()
+
+    def test_attendance_rank_for_committee(self):
+        return True
```
Comment (jbothma) [inline tests/unit/test_attendance.py]: WIP?

File: tests/unit/test_attendance.py [hunk 4]
```diff
@@ -0,0 +1,97 @@
+from tests import PMGTestCase
+from pmg.models import db, CommitteeMeeting, CommitteeMeetingAttendance, Committee, House, Province, Party, Member
+
+
+class TestCommitteeMeetingAttendance(PMGTestCase):
```
Comment (jbothma) [inline tests/unit/test_attendance.py]: Let's document:
This is testing:
- new data will show up on the page requesting "current" and NOT on "historical
- old data will show up on the page requesting "historical" and NOT on "current"

**Pull request merged**